src/ssvc/decision_points/human_impact.py

```diff
@@ -21,40 +24,59 @@
     description="Mission Prevalence:Minimal AND Public Well-Being Impact:Minimal",
 )
 
+LOW_2 = SsvcDecisionPointValue(
+    name="Low",
+    key="L",
+    description="Safety Impact:(None OR Minor) AND Mission Impact:(None OR Degraded OR Crippled)",
+)
+
+LOW_3 = SsvcDecisionPointValue(
+    name="Low",
+    key="L",
+    description="Safety Impact:(Negligible) AND Mission Impact:(None OR Degraded OR Crippled)",
+)
+
 MEDIUM_1 = SsvcDecisionPointValue(
     name="Medium",
     key="M",
     description="Mission Prevalence:Support AND Public Well-Being Impact:(Minimal OR Material)",
 )
 
+MEDIUM_2 = SsvcDecisionPointValue(
+    name="Medium",
+    key="M",
+    description="(Safety Impact:(None OR Minor) AND Mission Impact:MEF Failure) OR (Safety Impact:Major AND Mission Impact:(None OR Degraded OR Crippled))",
+)
+
+MEDIUM_3 = SsvcDecisionPointValue(
+    name="Medium",
+    key="M",
+    description="(Safety Impact:Negligible AND Mission Impact:MEF Failure) OR (Safety Impact:Marginal AND Mission Impact:(None OR Degraded OR Crippled))",
+)
+
 HIGH_1 = SsvcDecisionPointValue(
     name="High",
     key="H",
     description="Mission Prevalence:Essential OR Public Well-Being Impact:(Irreversible)",
 )
 
-VERY_HIGH_1 = SsvcDecisionPointValue(
```

Comment:
This was just reorganizing the file to group LOWs with LOWs, HIGHs with HIGHs, etc.